weaviate/templates/_helpers.tpl

```diff
@@ -196,8 +205,15 @@ Raft cluster configuration settings
             value: "{{ join "," $nodes }}"
   {{- else -}}
     {{- $votersCount := len (split "," .Values.env.RAFT_JOIN) -}}
+    {{- if empty .Values.env.RAFT_BOOTSTRAP_EXPECT }}
+      {{- $voters = $votersCount -}}
+    {{- end -}}
     {{- if not (eq $votersCount $voters)  -}}
       {{- fail "env.RAFT_BOOTSTRAP_EXPECT value needs to be equal to number of env.RAFT_JOIN nodes" -}}
     {{- end -}}
   {{- end -}}
+  {{- if empty .Values.env.RAFT_BOOTSTRAP_EXPECT }}
```

Comment:
I'm probably missing something, but this if conditional gives me the impression that if we pass a value for env.RAFT_BOOTSTRAP_EXPECT we will never define the RAFT_BOOTSTRAP_EXPECT variable...although checking the tests it looks like it does...but from the code if this is empty, we won't set a value..could you explain please?

Reply:
I will double check this, but I'm passing `env.RAFT_BOOTSTRAP_EXPECT` variable at runtime in the tests and then this block is not triggered and helm takes the value that was passed during runtime, but I will double check this.